tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_ceph_credentials.py

```
@@ -0,0 +1,192 @@
+# -*- coding: utf8 -*-
```

Ruff (0.16.1)

[warning] UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)